scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -105,7 +105,7 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         targetdevprefix = "sd"
         targetbus = "ide"
     controller_raid_volumes = args.controller_raid_volumes
-    if args.nodecounter != "1":
+    if args.nodecounter > args.numcontrollers:
```

Comment:
if there is not a default value this can fail, if I am not wrong, also make sure that you are comparing integers here